src/components/tabs/Tabs.js

```diff
@@ -55,44 +53,6 @@ export default class TabsComponent extends NestedComponent {
     return schema;
   }
 
-  build(state, showLabel) {
-    if (this.options.flatten) {
```

Comment:
Why was this logic removed? We need this for PDF generation.

Reply:
Sorry. This fix was initially for our project where this logic doesn't need. Fixed it